File: Java PowerPoint Examples/ThemeApiDemo.java
import spire.cloud.powerpoint.sdk.*;
import spire.cloud.powerpoint.sdk.api.ThemeApi;
import spire.cloud.powerpoint.sdk.model.*;

public class ThemeApiDemo {
    static String appId = "your id";
    static String appKey = "your key";
    static String baseUrl = "https://api.e-iceblue.cn";
    static Configuration configuration = new Configuration(appId, appKey, baseUrl);
    static ThemeApi themeApi = new ThemeApi(configuration);

    public static void getSlidesTheme() throws ApiException {
        String name = "template.pptx";
        Integer  slideIndex = 0;
        String password = null;
        String folder = "input";
        String storage = null;
        Theme response = themeApi.getSlidesTheme(name, slideIndex, password, folder, storage);
    }
    public static void getSlidesThemeColorScheme() throws ApiException {
        String name = "template.pptx";
        Integer  slideIndex = 0;
        String password = null;
        String folder = "input";
        String storage = null;
        ColorScheme response = themeApi.getSlidesThemeColorScheme(name, slideIndex, password, folder, storage);
    }
    public static void getSlidesThemeFontScheme() throws ApiException {
        String name = "template.pptx";
        Integer  slideIndex = 0;
        String password = null;
        String folder = "input";
        String storage = null;
        FontScheme response = themeApi.getSlidesThemeFontScheme(name, slideIndex, password, folder, storage);
    }
    public static void getSlidesThemeFormatScheme() throws ApiException {
        String name = "template.pptx";
        Integer  slideIndex = 0;
        String password = null;
        String folder = "input";
        String storage = null;
        FormatScheme response = themeApi.getSlidesThemeFormatScheme(name, slideIndex, password, folder, storage);
    }
}

File: C# PowerPoint Examples/ThemeApiDemo.cs
using Spire.Cloud.Powerpoint.Sdk.Client;
using Spire.Cloud.Powerpoint.Sdk.Api;

namespace CloudPowerpoint
{
    class ThemeApiDemo
    {
        static string appId = "your id";
        static string appKey = "your key";
        static string baseUrl = "https://api.e-iceblue.cn";
        static Configuration configuration = new Configuration(appId, appKey, baseUrl);
        static ThemeApi themeApi = new ThemeApi(configuration);
        public static void GetSlidesTheme()
        {
            string name = "template.pptx";
            int? slideIndex = 0;
            string password = null;
            string folder = "input";
            string storage = null;
            var response = themeApi.GetSlidesTheme(name, slideIndex, password, folder, storage);
        }      
        public static void GetSlidesThemeColorScheme()
        {
            string name = "template.pptx";
            int? slideIndex = 0;
            string password = null;
            string folder = "input";
            string storage = null;
            var response = themeApi.GetSlidesThemeColorScheme(name, slideIndex, password, folder, storage);
        }
        public static void GetSlidesThemeFontScheme()
        {
            string name = "template.pptx";
            int? slideIndex = 0;
            string password = null;
            string folder = "input";
            string storage = null;         
            var response = themeApi.GetSlidesThemeFontScheme(name, slideIndex, password, folder, storage);
        }
        public static void GetSlidesThemeFormatScheme()
        {
            string name = "template.pptx";
            int? slideIndex = 0;
            string password = null;
            string folder = "input";
            string storage = null;
            var response = themeApi.GetSlidesThemeFormatScheme(name, slideIndex, password, folder, storage);
        }

    }
}
